google-cloud-datastore/lib/google/cloud/datastore/v1/datastore_client.rb

```diff
@@ -93,14 +103,38 @@ def initialize \
             require "google/gax/grpc"
             require "google/datastore/v1/datastore_services_pb"
 
+            if channel || chan_creds || updater_proc
+              warn "The `channel`, `chan_creds`, and `updater_proc` parameters will be removed " \
+                "on 2017/09/08"
```

Comment:
I saw this output while running tests and it made me wonder about the date, which is already past... Does the date need to be updated?

Reply:
I'm going to remove these parameters from the code generator. It should propagate out to the GAPIC clients on the next refresh.